### DISCUSSION TITLE
对于“扭蛋收藏者纹章”的改动

### DISCUSSION THREAD

**Discussion by AviciiLove-Github** [ET-Team/EnigTech2#676] [Ideas]
原本的代码：
`mods.extendedcrafting.TableCrafting.addShaped(4, <contenttweaker:tlm_ultra>, [
	[<contenttweaker:auriminium>, null, null, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>, <touhou_little_maid:gashapon_coin>, <contenttweaker:draconium_alloy_ingot>, <contenttweaker:draconium_alloy_ingot>], 
	[null, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:hakurei_reimu_2"}), <contenttweaker:entropy>, null, null, <mekanismgenerators:reactorglass:1>, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>, <contenttweaker:draconium_alloy_ingot>], 
	[null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:kochiya_sanae"}), <contenttweaker:entropy>, <contenttweaker:entropy>, null, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>], 
	[null, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:izayoi_sakuya"}), <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:komeiji_satori"}), <contenttweaker:entropy>, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>], 
	[<mekanismgenerators:reactorglass:1>, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:alice_margatroid"}), <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:yukari_yakumo"}), <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:remilia_scarlet"}), <contenttweaker:entropy>, null, <mekanismgenerators:reactorglass:1>], 
	[<touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:komeiji_koishi"}), <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:kirisame_marisa"}), <contenttweaker:entropy>, null, null], 
	[<touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, null, <contenttweaker:entropy>, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:saigyouji_yuyuko"}), <contenttweaker:entropy>, null], 
	[<contenttweaker:draconium_alloy_ingot>, <touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, <mekanismgenerators:reactorglass:1>, null, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({ModelData: "touhou_little_maid:hakurei_reimu"}), null], 
	[<contenttweaker:draconium_alloy_ingot>, <contenttweaker:draconium_alloy_ingot>, <touhou_little_maid:gashapon_coin>, <touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, null, null, <contenttweaker:auriminium>]
]);`
但是在jei中显示出错，且无法查清模型
可更改为
`mods.extendedcrafting.TableCrafting.addShaped(4, <contenttweaker:tlm_ultra>, [
	[<contenttweaker:auriminium>, null, null, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>, <touhou_little_maid:gashapon_coin>, <contenttweaker:draconium_alloy_ingot>, <contenttweaker:draconium_alloy_ingot>], 
	[null, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:hakurei_reimu_2"}), <contenttweaker:entropy>, null, null, <mekanismgenerators:reactorglass:1>, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>, <contenttweaker:draconium_alloy_ingot>], 
	[null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:kochiya_sanae"}), <contenttweaker:entropy>, <contenttweaker:entropy>, null, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>], 
	[null, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:izayoi_sakuya"}), <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:komeiji_satori"}), <contenttweaker:entropy>, null, <mekanismgenerators:reactorglass:1>, <touhou_little_maid:gashapon_coin>], 
	[<mekanismgenerators:reactorglass:1>, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:alice_margatroid"}), <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:yukari_yakumo"}), <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:remilia_scarlet"}), <contenttweaker:entropy>, null, <mekanismgenerators:reactorglass:1>], 
	[<touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:komeiji_koishi"}), <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:kirisame_marisa"}), <contenttweaker:entropy>, null, null], 
	[<touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, null, <contenttweaker:entropy>, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:saigyouji_yuyuko"}), <contenttweaker:entropy>, null], 
	[<contenttweaker:draconium_alloy_ingot>, <touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, <mekanismgenerators:reactorglass:1>, null, null, <contenttweaker:entropy>, <touhou_little_maid:maid_model_coupon:5>.withTag({"com.github.tartaricacid.touhoulittlemaid.client.resources.ModelData": "touhou_little_maid:hakurei_reimu"}), null], 
	[<contenttweaker:draconium_alloy_ingot>, <contenttweaker:draconium_alloy_ingot>, <touhou_little_maid:gashapon_coin>, <touhou_little_maid:gashapon_coin>, <mekanismgenerators:reactorglass:1>, null, null, null, <contenttweaker:auriminium>]
]);`

**Comment by dong031001** [comment]
下个版本考虑更改，感谢建议！
PS: 如果直接对代码进行改动，或许发一个PR会更好。
